demo/grid-pro-edit-column-demos.html

```diff
@@ -192,7 +192,7 @@ <h3>Prevent Saving Change</h3>
     <h3>Custom Edit Mode Renderer</h3>
     <p>
       The custom editor component can be rendered for cell edit mode using <b><code>editModeRenderer</code></b>
-      function. Please make sure to render the component as a first child to the <code>root</code> provided as a first
+      function. Please make sure to render the component as a light DOM child to the <code>root</code> provided as a first
```

Comment:
I would recommend removing "light DOM" part from it, and use "child" instead. That should be clear enough.

Reply:
Updated.